frontendservice/src/views/ShopOwnerHome.vue

```diff
@@ -1,79 +1,169 @@
-under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
-CONDITIONS OF ANY KIND, either express or implied. See the License for the
-specific language governing permissions and limitations under the License. */
+/*
+Copyright 2020 Google LLC
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    https://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
 
 <template>
-  <div class="home">
-    <h1 v-if="isLoggedIn">Welcome! {{ name }}</h1>
-    <p>
-      This page will be Home page for the shop-owners. 
-    </p>
-    <p>
-      ToDo: Implement ShopOwnerHome page features including NavBar, Search Bar, MyShops and AddShop.
-    </p> 
+  <div>
     <div v-if="isVerified">
-      <router-link to="/create-shop">Create new shop</router-link>
-    </div>
-    <div v-if="isVerified">
-      <router-link to="/my-shops">My Shops</router-link>
+      <shopOwnerNavbar />
+      <h1 id="heading" v-if="isVerified">My Shops</h1>
+      <input type="search" v-model="searchResult" placeholder="search shops" id="searchbar" />
+      <table class="table">
+        <thead>
+          <th>Shop Name</th>
+          <th>Shop Type</th>
+          <th>All Tokens of Shop</th>
+          <th>Delete</th>
+        </thead>
+        <tbody>
+          <tr v-for="shop in filteredShops" :key="shop['shop'].shopId">
+            <td>
+              <a v-bind:href="'/shop-owner/shops/'+ shop['shop'].shopId" :shopId="shop['shop'].shopId">{{shop['shop'].shopName}}</a>
+            </td>
+            <td>{{shop['shop'].shopType}}</td>
+            <td v-if="shop['shop'].status=='ACTIVE'">
+              <a v-bind:href="'/shop-owner/shops/'+ shop.shopId" :id="shop.shopId">{{shop.numberOfActiveTokens}}</a>
+            </td>
+            <td v-if="shop['shop'].status!='ACTIVE'">
+              <a v-bind:href="'/shop-owner/shops/'+ shop.shopId" :id="shop.shopId">0</a>
+            </td>
+            <td v-if="shop['shop'].status=='ACTIVE'">
+              <button @click="deleteshop(shop['shop'].shopId)">x</button>
+            </td>
+            <td v-if="shop['shop'].status!='ACTIVE'">
+              <button>restore</button>
+            </td>
+          </tr>
+        </tbody>
+      </table>
     </div>
     <div v-if="isLoggedIn && !isVerified">
-      <router-link to="/verify">Verify Me!</router-link>
-    </div>
-    <div v-if="!isLoggedIn">
-      <router-link to="/about">Back</router-link>
+      <router-link to="/verify"><button class="button is-primary">Verify Me!</button></router-link>
```

Comment:
Can add a ToDo here to add other options as well in case of "v-if="isLoggedIn && !isVerified"" like Logout.

Reply:
Added todo here as well as in create shop.